data/config.yaml

```diff
@@ -1,8 +1,8 @@
 _id: d779730c7cc94155877a24e2e8be2ad1
-created: '2022-04-18T14:36:30.754421'
+created: '2022-10-28T15:23:03.285808'
```

Comment:
Note really part of this PR, but we really need to decide what to do with this file - the tests should not be changing it, and it definitely shouldn't be something that shows up in PRs.